ctapipe/image/tests/test_sliding_window_correction.py

```diff
@@ -0,0 +1,69 @@
+"""
+Test of sliding window extractor for LST camera pulse shape with
+the correction for the integration window completeness
+"""
+
+import numpy as np
+import astropy.units as u
+from numpy.testing import assert_allclose
+from traitlets.config.loader import Config
+
+from ctapipe.image.extractor import SlidingWindowMaxSum, ImageExtractor
+from ctapipe.image.toymodel import WaveformModel
+from ctapipe.instrument import SubarrayDescription, TelescopeDescription
+
+from ctapipe.utils import datasets
+
+
+def test_sw_pulse_lst(monkeypatch):
+    """
+    Test function of sliding window extractor for LST camera pulse shape with
+the correction for the integration window completeness
+    """
+
+    with monkeypatch.context() as monkey_patch:
```

Comment:
This can be removed now that the other PR is merged.

To update: 
```
git fetch upstream
git merge upstream/master
```

Reply:
@maxnoe done